test_script.py

```diff
@@ -1,12 +1,12 @@
 import time
-import sys
 from hyperdash.sdk import monitor
 
 
-@monitor(job_name="ml_model")
+@monitor(job_name="cats v. dogs")
```

Comment:
I think we're calling the function monitor and it will be in the hyperdash namespace